test/angular-material-spec.js

```diff
@@ -194,10 +200,71 @@
             return results;
           }
         };
-      }
+      },
+
+      /**
+       * Asserts that a given selector matches#006b75 one or more items.
+       * Accepts any of:
+       *   {string} - A CSS selector.
+       *   {angular.JQLite} - The result of a jQuery query.
+       *   {Element} - A DOM element.
+       */
+      toExist: function() {
+        return {
+          compare: function(actual) {
+            var el = getElement(actual);
+            var pass = el.length > 0;
+            var not = pass ? 'not ' : '';
+
+            return {
+              pass: pass,
+              message: 'Expected "' + actual +
+              '" ' + not + 'to match element(s), ' +
+              'but found ' + el.length +
+              ' items in the DOM'
+            };
+          }
+        };
+      },
+
+      /**
+       * Asserts that a given element contains a given substring in
+       * its innerHTML property.
+       * Accepts any of:
+       *   {string} - A CSS selector.
+       *   {angular.JQLite} - The result of a jQuery query.
+       *   {Element} - A DOM element.
+       */
+      toContainHtml: function() {
```

Comment:
I believe I these methods will be useful, and am adding all of them now (instead of just the `toHaveClass` method changes) so later down the line we don't get merge conflicts when we merge feat/panel.